File: sort/LinearSearch.py
data = [2, 45, 23, 12, 9, 56, 3, 23, 10, 32, 12]

def linear_search(data, num):
    for i in range(len(data)):
        if num == data[i]:
            print(f"Number found at index {i}")

linear_search(data, 12)

File: search/LinearSearch.java
package search;

public class LinearSearch {
    public static void main(String[] args) {
        int[] data = {43, 21, 0, 32, 4, 32, 54, 2, 10, 11};
        int num = 32;

        for(int i = 0; i < data.length; i++){
            if(num == data[i]){
                System.out.println("Number found at index " + i);
            }
        }
    }
}
